src/Controller/Backend/General.php

```diff
@@ -197,15 +197,17 @@ public function prefill(Request $request)
 
         if ($request->isMethod('POST') || $request->query->getBoolean('force')) {
             $form->handleRequest($request);
-            if ($form->get('contenttypes')->has('contenttypes')) {
+            if (!empty($form->get('contenttypes')->getData())) {
```

Comment:
It returns `[]` when nothing is selected. Open to suggestions on how to do it properly.